common/src/main/java/com/gentics/mesh/core/data/Branch.java

```diff
@@ -432,18 +432,22 @@ default TypeInfo getTypeInfo() {
 	 * @param schemaContainerVersion
 	 * @param assigned
 	 * @param status
+	 * @param oldVersion old version (may be null)
 	 * @return
 	 */
-	BranchSchemaAssignEventModel onSchemaAssignEvent(SchemaContainerVersion schemaContainerVersion, Assignment assigned, JobStatus status);
+	BranchSchemaAssignEventModel onSchemaAssignEvent(SchemaContainerVersion schemaContainerVersion, Assignment assigned,
```

Comment:
It seems like this method is extended from `HibBranch`. Could you change the signature there? It would make things easier when we will port this change to mdm. Same is true for `onMicroschemaAssignEvent`

Reply:
`HibBranch` does not exist in this branch